src/file/mod.rs

```diff
@@ -243,7 +243,7 @@ impl TempPath {
         }
     }
 
-    /// Persist the temporary file at the target path iff no file exists there.
+    /// Persist the temporary file at the target path if no file exists there.
```

Comment:
This is actually intentional. Iff means "if and only if". However, you're not the first to point this out so it's probably better to just expand the phrase.

```suggestion
    /// Persist the temporary file at the target path if and only if no file exists there.
```

Reply:
That makes sense! The rest of the documentation is really well written, so I thought a typo like that was really out of place. Thank you for the clarification :)

Reply:
Well, the "it's" was definitely wrong and a bit embarrassing. Thanks for catching it!

(and iff needed to go anyways)